src/nnue/evaluate_nnue.cpp

```diff
@@ -70,14 +70,22 @@ namespace Eval::NNUE {
     std::memset(pointer.get(), 0, sizeof(T));
   }
 
+  template <typename T>
+  void Initialize(TtmemPtr<T>& pointer) {
+
+    static_assert(alignof(T) <= 4096, "Ensure aligned_ttmem_alloc gives big enough alignment");
+    pointer.reset(reinterpret_cast<T*>(aligned_ttmem_alloc(sizeof(T))));
+    std::memset(pointer.get(), 0, sizeof(T));
+  }
+
   // Read evaluation function parameters
   template <typename T>
-  bool ReadParameters(std::istream& stream, const AlignedPtr<T>& pointer) {
+  bool ReadParameters(std::istream& stream, T& reference) {
```

Comment:
Note: This is the only change I made in the force push (in addition to commit msg change for STC):

Old code: T &reference 
New code: T& reference